composer.json

```diff
@@ -1,5 +1,5 @@
 {
-    "name": "mindplay/annotations",
+    "name": "Naerymdan/annotations",
```

Comment:
Can't you just specify different repo in composer.json via `repositories` key and then your fork will be used to get original package.